### PR TITLE
Adding Town Director

### PR DESCRIPTION
**Pull request by Malerus**
## About The Pull Request
Adding Town Director as requested from playerbase

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: Adds Town Director role
/:cl:

### CODE REVIEW

**Comment by ErwinSnowden** [comment]
You should give them the cyberneticist expert trait and better surgery than the regular doctors.

**Comment by Malerus** [comment]
> You should give them the cyberneticist expert trait and better surgery than the regular doctors.

Done

**Comment by BadAtThisGame302** [comment]
Why give the CMO his own access when there's nothing that uses it at all?

**Comment by Malerus** [comment]
> Why give the CMO his own access when there's nothing that uses it at all?

It's for job assignment on the ID console for the mayor and for their own office.

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -58,7 +58,7 @@
 	var/damage_deflection = 10
 	resistance_flags = FIRE_PROOF
 	armor = ARMOR_VALUE_MEDIUM
-	req_access = list(ACCESS_ENGINE_EQUIP, ACCESS_BOS)
+	req_access = list(ACCESS_ENGINE_EQUIP, , ACCESS_TOWN_SCIENCE)
```
Comment (OliOliOnsiPree) [inline code/modules/power/apc.dm:61]: The bos has apcs in there base, unless they are specific ones.

Comment (Malerus) [reply code/modules/power/apc.dm:61]: > The bos has apcs in there base, unless they are specific ones.

The basement clinic has APCs too

**Comment by cornercube** [comment]
merged some features into #322

**Comment by companoo** [comment]
delegator not deligator

**Pull request closed without merging**